lib/rules/disallow-space-after-object-keys.js

```diff
@@ -41,7 +41,7 @@ module.exports.prototype = {
     check: function(file, errors) {
         file.iterateNodesByType('ObjectExpression', function(node) {
             node.properties.forEach(function(property) {
-                if (property.shorthand) {
+                if (property.shorthand || property.method) {
```

Comment:
I very like that idea but realization can be painful. Anyway, we can discuss about it and decide in that issue ;-) So, yep!